src/components/menu/menu-trigger/menu-trigger.directive.ts

```diff
@@ -236,6 +236,14 @@ export class MenuTriggerDirective implements OnInit, OnDestroy {
         }
     }
 
+    @HostListener('mousemove')
+    _onMouseMove(): void {
+        if (this._isSubmenuTrigger && !this._parentMenu._isAnimating) {
+            timer(this._debounceTime).pipe(takeUntil(this._onDestroy$))
```

Comment:
Does `setTimeout` work here?

Reply:
It does indeed, I will push up the change now